test/fail_compilation/fail21885.d

```diff
@@ -0,0 +1,25 @@
+// https://issues.dlang.org/show_bug.cgi?id=21885
+
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail21885.d(24): Error: struct `fail21885.Outer` is not copyable because field `i` is not copyable
```

Comment:
Can we recurse, so that "because it has a disabled postblit" is shown as supplemental ?

Reply:
Sorry, I don't understand what you mean. Do you expect here to have:

```
Error: struct `fail21885.Outer` is not copyable because it has a disabled postblit.
          The postblit is disabled because field `i' is not copyable and no user defined postblit was provided.
```

?

Reply:
```D
struct A { B b; }
struct B { C c; }
struct C { int a; @disable this(this); }
void foo (A a);
```

I mean that this should produce, if possible:
```
file.d:42: Error: struct `mod.A` is not copyable because field `b` is not copyable.
                           struct `mod.B` is not copyable because field `c` is not copyable
                           struct `mod.C` is not copyable because it has a disabled postblit
```

Reply:
I see. I think that this could be implemented behind a switch as for large structs the output could be quite verbose. Anyway, I would rather get this in and file your proposal as an enhancement request.